<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MbulbWindow</class>
 <widget class="QDialog" name="MbulbWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mandelbulb Generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="hL_presets" stretch="0,0,0,0">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Preset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_presets"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hL_newPreset">
         <item>
          <widget class="QLineEdit" name="lineEdit_newPresetName">
           <property name="placeholderText">
            <string>preset name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonNewPreset">
           <property name="toolTip">
            <string>Create new preset</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/cil-plus.svg</normaloff>:/icons/cil-plus.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonDeletePreset">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/cil-trash.svg</normaloff>:/icons/cil-trash.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hL_volumeInput" stretch="0,0,0">
       <item>
        <layout class="QVBoxLayout" name="vL_pointAInput">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>X1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="X1Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetX1">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Y1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Y1Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetY1">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Z1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Z1Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetZ1">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_equalValues_P1">
           <property name="text">
            <string>Equal values</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vL_pointAInput_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>X2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="X2Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>-1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetX2">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Y2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Y2Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>-1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetY2">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Z2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Z2Input">
             <property name="minimum">
              <double>-5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>5.000000000000000</double>
             </property>
             <property name="value">
              <double>-1.750000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetZ2">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_equalValues_P2">
           <property name="text">
            <string>Equal values</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <layout class="QVBoxLayout" name="vL_parameters">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Iterations</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="IterInput">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>32767</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetIter">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Power</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="powerInput">
             <property name="maximum">
              <double>16383.000000000000000</double>
             </property>
             <property name="value">
              <double>8.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetPower">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Max. Length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="maxLengthInput">
             <property name="maximum">
              <double>16383.000000000000000</double>
             </property>
             <property name="value">
              <double>2.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetMaxLength">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>x-Resolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="xresInput">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>32767</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetXres">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>y-Resolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="yresInput">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>32767</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetYres">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>z-Resolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="zresInput">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>32767</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonResetZres">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/cil-reload.svg</normaloff>:/icons/cil-reload.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_equalValues_res">
           <property name="text">
            <string>Equal values</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>x-Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="xdistOutput">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>y-Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ydistOutput">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>z-Distance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="zdistOutput">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Grid Slots</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gridSlotsOutput">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Grid points</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pointCountOutput">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vL_options" stretch="0">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonGenerate">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
